mapsTask/js/controller.js

```diff
@@ -0,0 +1,81 @@
+
+$(document).ready(function () {
+    $("#name").focus();
```

Comment:
We can use the ` autofocus ` html attribute of the input field, instead of handling it using JavaScript.
Take a look at an example: 
`<input type="text" name="fname" autofocus>`